uk.co.compucorp.civicrm.hrleaveandabsences/xml/schema/CRM/HRLeaveAndAbsences/PublicHoliday.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="iso-8859-1" ?>
+
+<table>
+  <base>CRM/HRLeaveAndAbsences</base>
+  <class>PublicHoliday</class>
+  <name>civicrm_hrleaveandabsences_public_holiday</name>
+  <comment>FIXME</comment>
+  <add>4.4</add>
+  <log>true</log>
+
+  <field>
+    <name>id</name>
+    <type>int unsigned</type>
+    <required>true</required>
+    <comment>Unique PublicHoliday ID</comment>
+    <add>4.4</add>
+  </field>
+  <primaryKey>
+    <name>id</name>
+    <autoincrement>true</autoincrement>
+  </primaryKey>
+
+  <field>
+    <name>title</name>
+    <title>Title</title>
+    <type>varchar</type>
+    <length>127</length>
+    <required>true</required>
+    <comment>The Public Holiday title</comment>
+  </field>
+  <field>
+    <name>date</name>
+    <title>Date</title>
+    <type>date</type>
+    <required>true</required>
+    <unique>true</unique>
```

Comment:
Was this enough for it to generate the correct UNIQUE index to the generated SQL in civicrm.mysql? Whenever I tried this it didn't work. I had to manually add the index to the schema like this: https://github.com/civicrm/civihr/blob/develop/uk.co.compucorp.civicrm.hrleaveandabsences/xml/schema/CRM/HRLeaveAndAbsences/WorkDay.xml#L98-L104